### PR TITLE
ddt removal in cli org tests

### PR DESCRIPTION
**Pull request by sthirugn**
This PR also:
* Removes closed bugzilla references
* Made pylint 100%
* optimized/renamed test data

### CODE REVIEW

**Comment by sthirugn** [comment]
Marking this `do not merge` as I am running the tests now.  I will remove `do not merge` flag once test results are available.

**Comment by sthirugn** [comment]
Test result: (the results match the latest jenkins run)
```sh
Finding files... done.
Importing test modules ... done.

----------------------------------------------------------------------
Ran 117 tests in 978.571s

OK (skipped=71)
```

**Comment by omaciel** [comment]
Cool!!! Do you know if the suite is running faster?

**Comment by sthirugn** [comment]
@omaciel I dont have the old metrics.

**Comment by omaciel** [comment]
@sthirugn from Jenkins perhaps?

**Comment by sthirugn** [comment]
@omaciel yeah I will check

**Comment by sthirugn** [comment]
@omaciel jenkins says `117 tests (±0) Took 10 min.`  But my local test says 978.571s or 16.3 minutes.  But jenkins runner and my box are of different capacity so as the sat6 box itself.  I will have to wait till next jekins run to see how these refactor affected test timing.

File: tests/foreman/cli/test_org.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # -*- encoding: utf-8 -*-
-# pylint: disable=R0904
+# pylint: disable=too-many-public-methods, too-many-lines
```
Comment (elyezer) [inline tests/foreman/cli/test_org.py]: :+1:

**Comment by elyezer** [comment]
Looks good to me. ACK

**Comment by sthirugn** [comment]
Rebased now

File: tests/foreman/cli/test_org.py [hunk 2]
```diff
@@ -691,25 +739,27 @@ def test_add_configtemplate(self, data):
...
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
-        result = Org.info({'id': org['id']})
-        self.assertIn(
-            u'{0} ({1})'. format(template['name'], template['type']),
-            result.stdout['templates']
-        )
+        for data in valid_names_simple_all():
+            with self.subTest(data):
```
Comment (omaciel) [inline tests/foreman/cli/test_org.py]: s/data/name?

**Comment by omaciel** [comment]
ok, will ACK it now and perhaps we can update the name of the variables used to represent the data being passed to `subTest` so that they are more meaninful.

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**